src/backup-all-indexes.sh

```diff
@@ -39,7 +55,17 @@ else
     REPOSITORY_PLUGIN=repository-s3
 fi
 
-backup_index ()
+
+# small slack notification function
+
+slack_send() { 
+
+  curl -XPOST --data-urlencode 'payload={"channel": "'"$SLACK_CHANNEL"'", "text": "'"$1"'", "username": "'"$SLACK_USER"'", "icon_emoji": "'"$2"'" }' $SLACK_HOOK
```

Comment:
Can we use a heredoc to make the payload somewhat readable? 

Here is an example: https://github.com/aptible/docker-rabbitmq/blob/d7374b0f424a115665b5607f9811424c1a88ac3c/bin/run-database.sh#L100-L108

Reply:
Also: 

- please quote `$SLACK_HOOK`.
- please make this `curl -fsS` to minimize noise in logs.